### PR TITLE
[Dashboard] Remove VStack - Need test so do not merge

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
The focus of this PR is to replace the `VStack` components with `div` elements in multiple files for better styling and layout control.

### Detailed summary
- Replaced `VStack` with `div` in `OnboardingTitle`, `JudgesEarn`, `AvatarShowcase`, and more components for styling.
- Updated import statements for `Flex` and `SimpleGrid` in `JudgesEarn` and `Judges` components.
- Replaced `VStack` with `div` in `ReceiveFundsModal`, `UploadStep`, and more components for consistent styling.
- Removed `VStack` import in `BillingPeriod` and `OnboardingGeneral` components.

> The following files were skipped due to too many changes: `apps/dashboard/src/components/onboarding/Steps.tsx`, `apps/dashboard/src/components/settings/Account/AccountForm.tsx`, `apps/dashboard/src/components/settings/Account/Usage.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4727?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4727** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4727?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4722** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4722?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/components/hackathon/HackathonFooter.tsx [hunk 1]
```diff
@@ -15,17 +15,7 @@ export const HackathonFooter = ({
...
-      bgSize="cover"
-      bgRepeat="no-repeat"
-      bgPosition="center"
-      w="100%"
-      py={20}
-      gap={10}
-      borderTopRadius="50px"
-      px={6}
-    >
+    <div className="flex flex-col items-center bg-[url('/assets/hackathon/footer-bg.png')] bg-cover bg-no-repeat bg-center w-full py-20 space-y-10 rounded-t-[50px] px-6">
```
Comment (jnsdls) [inline apps/dashboard/src/components/hackathon/HackathonFooter.tsx:18]: prefer `gap` over `space`

File: apps/dashboard/src/components/hackathon/Judges.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ export const Judges = ({ TRACKING_CATEGORY }: JudgesProps) => {
   ];
 
   return (
-    <VStack spacing={8} position="relative">
+    <div className="space-y-8 relative flex flex-col">
```
Comment (jnsdls) [inline apps/dashboard/src/components/hackathon/Judges.tsx:38]: gap over space

File: apps/dashboard/src/components/hackathon/JudgesEarn.tsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ export const JudgesEarn = () => {
   ];
 
   return (
-    <VStack spacing={8} position="relative">
+    <div className="flex flex-col space-y-8 relative">
```
Comment (jnsdls) [inline apps/dashboard/src/components/hackathon/JudgesEarn.tsx:30]: gap

File: apps/dashboard/src/components/hackathon/common/AvatarShowcase.tsx [hunk 4]
```diff
@@ -18,7 +18,7 @@ export const AvatarShowcase: React.FC<AvatarShowcaseProps> = ({
   trackingCategory,
 }) => {
   return (
-    <VStack spacing={12}>
+    <div className="flex flex-col spacing-y-12">
```
Comment (jnsdls) [inline apps/dashboard/src/components/hackathon/common/AvatarShowcase.tsx:21]: mind the gap

File: apps/dashboard/src/components/onboarding/Steps.tsx [hunk 5]
```diff
@@ -273,11 +273,8 @@ export const OnboardingSteps: React.FC<OnboardingStepsProps> = ({
       overflow="hidden"
       className="bg-muted/50"
     >
-      <VStack
-        gap={2}
-        alignItems="flex-start"
-        p={6}
-        w={rightImageDark && !isMobile ? "60%" : "100%"}
+      <div
+        className={`gap-2 flex flex-col items-start p-6 ${rightImageDark && !isMobile ? "w-[60%]" : "w-full"}`}
```
Comment (jnsdls) [inline apps/dashboard/src/components/onboarding/Steps.tsx:277]: use `cn()` util for optional classes

File: apps/dashboard/src/components/settings/Account/AccountForm.tsx [hunk 6]
```diff
@@ -138,15 +130,8 @@ export const AccountForm: React.FC<AccountFormProps> = ({
...
-        alignItems="flex-start"
-        w="full"
-        gap={horizontal ? 6 : 8}
-        borderRadius="xl"
-        borderColor="borderColor"
-        borderWidth={padded ? 1 : 0}
-        bg={bg}
-        p={padded ? 6 : 0}
+      <div
+        className={`items-start w-full ${horizontal ? "gap-6" : "gap-8"} rounded-xl ${padded ? "border border-border p-6" : "p-0"} bg-background`}
```
Comment (jnsdls) [inline apps/dashboard/src/components/settings/Account/AccountForm.tsx:134]: `cn()` for conditionals

**Review by jnsdls — CHANGES_REQUESTED** [review]

**Pull request closed without merging**